NetUtils.py

```diff
@@ -11,12 +11,14 @@
 from Utils import ByValue, Version
 
 
-class HintStatus(ByValue, enum.IntEnum):
-    HINT_UNSPECIFIED = 0
-    HINT_NO_PRIORITY = 10
-    HINT_AVOID = 20
-    HINT_PRIORITY = 30
-    HINT_FOUND = 40
+class HintPriority(ByValue, enum.IntEnum):
+    HINT_UNSPECIFIED = 0     # No priority has been specified yet
+    HINT_CAN_SKIP = 10       # The item can be skipped
+    HINT_MIGHT_WANT = 20     # The player is unsure if they want the item or not
+    HINT_AVOID = 30          # The item should be avoided if at all possible
+    HINT_AVOID_FOR_NOW = 40  # The item should be avoided for now, but may be needed later
```

Comment:
Why a distinction between "avoid" and "avoid for now"? It's always possible to change the status from "avoid" to "don't avoid" at any time, the time-boundedness of the status is inherently captured by the ability to change it.

Reply:
Implication. If you set something to `Avoid`, while `Avoid For Now` exists, you're implying `I want you to avoid this and have no intention at this time of changing that`. Meanwhile, some things, namely prog+traps like Terraria's `Hardmode` or Stardew's `Progressive Movie Theater`, are things that you tend to want avoided, but may eventually NEED for progress- thus, inherently, something that you would want to avoid *for a short time*, and then get later.

As the point of these priorities is to foster communication, I feel the additional priority to handle that case makes enough sense.

Reply:
Then should the "for now" also be extended to other priority types? Vi's flag pr may help here

Reply:
I mean... you COULD, but I don't see it as anywhere near as necessary for anything aside from `Avoid`. `Avoid` as a priority states `Do not get this at all costs`, and as such is unique from other priorities (it's the only priority that actively wants you to avoid an item).

Honestly at this point I don't have the energy for this, so, whoever else wants to do anything with any of this is welcome to. (As I already told Vi)